src/renderer_vk.cpp

```diff
@@ -1345,7 +1348,7 @@ VK_IMPORT_INSTANCE
 				dci.ppEnabledLayerNames  = enabledLayerNames;
 				dci.enabledExtensionCount   = numEnabledExtensions;
 				dci.ppEnabledExtensionNames = enabledExtension;
-				dci.pEnabledFeatures = NULL;
+				dci.pEnabledFeatures = &m_deviceFeatures;
```

Comment:
Passing the output of `vkGetPhysicalDeviceFeatures` directly like this will unnecessary enable e.g. `robustBufferAccess` if available, which has a performance impact. I'd consider turning that off explicitly here. :)

Reply:
Thanks for pointing!

Reply:
@cforfang Then how about this on line 1160?
```c++
VkPhysicalDeviceFeatures deviceFeatures;
vkGetPhysicalDeviceFeatures(m_physicalDevice, &deviceFeatures);
bx::memSet(&m_deviceFeatures, 0, sizeof(VkPhysicalDeviceFeatures));
m_deviceFeatures.independentBlend = deviceFeatures.independentBlend;
```

Reply:
While I _think_ that would work, I'm not 100% if not enabling things like `textureCompressionETC2` / `textureCompressionASTC_LDR` / `textureCompressionBC` might either (a) literally turn off support in the drivers, or (b) cause validation layers to complain.

It might be simpler to just enable everything available except `robustBufferAccess` as I don't know of any other features with a performance impact.